vulture/lines.py

```diff
@@ -1,12 +1,14 @@
 import ast
-from functools import wraps
+from collections import namedtuple
 import sys
 
+ENTRY_POINTS = (ast.ClassDef, ast.FunctionDef)
```

Comment:
These entry points were only for testing. The code should work for all types of nodes.